docs/pages/accounts/sso.mdx

```diff
@@ -21,6 +26,34 @@ Expo SSO supports the following identity providers:
 
 We implement the [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html) specification and are working to verify additional compatible identity providers. If you use another identity provider and are interested in SSO, [let us know](https://expo.dev/contact).
 
+## Enable SSO
+SSO is enabled by creating SSO configuration for the Organization.
+
+<Step label="1">
+	Log in as the Organization account owner, select your Organization Account, then go to Organization Settings > Overview.
```

Comment:
```suggestion
	Log in as the Organization account owner, select your Organization Account, then go to **Organization Settings** > **Overview**.
```

Let's prefer highlighting the Settings name in expo.dev (as we do in the rest of the docs).

Reply:
Done